views/home.njk

```diff
@@ -117,7 +117,7 @@
         <div class="usa-width-two-thirds description">
           <h2>Launch your site in minutes.</h2>
           <p class="copy">
-            Is your code already on GitHub? Federalist can securely deploy a static website from your repository in minutes. Federalist also deploys site previews from GitHub branches, allowing your team to easily prototype site updates.
+            Federalist uses [GitHub](https://github.com/) as our software development platform. If your code is already on GitHub, Federalist can securely deploy a static website from your repository in minutes. 
```

Comment:
Should we say GitHub or just that Federalist loads site code from a public code hosting service - and then link to Github?

Reply:
Yup! Nice catch. Will edit.